legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/contract/storeContract.pure

```diff
@@ -166,6 +175,8 @@ function meta::relational::contract::supports(f:FunctionExpression[1]):Boolean[1
             meta::pure::functions::math::sum_Float_MANY__Float_1_,
             meta::pure::functions::math::sum_Integer_MANY__Integer_1_,
             meta::pure::functions::math::sum_Number_MANY__Number_1_,
+            // TODO HSO: Where does this belong?
+            meta::pure::mutation::execution::save_Class_1__RootGraphFetchTree_1__T_MANY__Mapping_1__Runtime_1__T_MANY_,
```

Comment:
I think this belong to any store that wants to support save, hence this seems correct to me.

Reply:
Moved into the getSupportedFunctions block